### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by sachhm**
@ShaneLind sorry for the late submission, but could you please take a look at my code? I'm still not 100 on the whole Association not Inheritance exercise. 

Cheers

### CODE REVIEW

File: prac_09/taxi_simulator.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        menu_choice = input("q)uit, c)hoose taxi, d)rive\n>>>").upper()
+
+    print("Taxis are now: ")
+    display_taxis(taxis)
+
+
+def display_taxis(taxis):
+    """Display taxis with index location in list"""
+    for i, taxi in enumerate(taxis):
+        print(f"{i} - {taxi}")
```
Comment (ShaneLind) [inline prac_09/taxi_simulator.py:44]: This is cool. Way better than making a counter increase. I'm stealing this for future use. Thanks.

File: prac_09/taxi_simulator.py
Comment (ShaneLind) [inline prac_09/taxi_simulator.py:1]: I believe you have forgotten to print total trip cost when the program quits.

**Comment by ShaneLind** [comment]
Your code was great. Everything you did was better than what I had done, so I apologise for the lack of comments. I've learned quite a few things from your code so thanks for that. Great work.

File: prac_09/taxi_simulator.py (hunk 1 above)
Comment (sachhm) [reply prac_09/taxi_simulator.py:44]: Haha no worries! I actually stole this off some body else's code that I reviewed. 😆

**Comment by sachhm** [comment]
Thanks for reviewing my code!

**Pull request merged**